lldb/test/API/commands/frame/var/anon-struct/main.cpp

```diff
@@ -0,0 +1,19 @@
+int main(int argc, char** argv) {
+  struct A {
+    struct {
+      int x = 1;
+    };
+    int y = 2;
+  } a;
+
+  struct B {
+    // Anonymous struct inherits another struct.
+    struct : public A {
+      int z = 3;
+    };
+    int w = 4;
+    A a;
+  } b;
```

Comment:
Can we avoid running the binary by making this a global variable (and then using `target variable` in the test case)?

Reply:
Done.